sycl/unittests/pi/PlatformTest.cpp

```diff
@@ -32,7 +32,9 @@ class PlatformTest : public ::testing::Test {
     // Initialize the logged number of platforms before the following assertion.
     RecordProperty(platform_count_key, platform_count);
 
-    ASSERT_EQ(RT::piPlatformsGet(0, 0, &platform_count), PI_SUCCESS);
+    ASSERT_EQ(RT::PluginInformation.PiFunctionTable.piPlatformsGet(
```

Comment:
Expecting a follow up change to move the plugin into a SYCL object, e.g., a platform (?) - should I look into moving the unit tests to use the `PI_CALL`-alike macros or should I wait until it is clear how the function pointer table is provided to the macros?

Reply:
I think I should change this now and appropriately later too when the pointers are moved into a SYCL object.

Reply:
Thank you!!!